lib/Widgets/DynamicNotebook.vala

```diff
@@ -933,11 +941,6 @@ namespace Granite.Widgets {
         }
 
         ~DynamicNotebook () {
```

Comment:
Any reason to keep this block now it is empty?

Reply:
No particular reason, I guess it doesn't hurt to keep it for future use.